src/lib/core/breakpoints/break-point.ts

```diff
@@ -10,4 +10,6 @@ export interface BreakPoint {
   alias: string;
   suffix?: string;
   overlapping?: boolean;
+  priority?: number;
+  includes?: string[];
```

Comment:
To be removed, but a possible design for breakpoint prioritization

Reply:
Let's add comments to document api adds.